src/radio/index.ts

```diff
@@ -164,7 +164,13 @@ export class Radio extends ThemedMixin(FocusMixin(WidgetBase))<RadioProperties>
 					ontouchstart: this._onTouchStart,
 					ontouchend: this._onTouchEnd,
 					ontouchcancel: this._onTouchCancel
-				})
+				}),
+				v('div', {
+					classes: this.theme(css.controlIndicator)
```

Comment:
what do you think about using `radioBackground`, `radioOuter`, `radioInner` class names?